src/peerbook/libp2p_peerbook.erl

```diff
@@ -306,7 +306,8 @@ mk_this_peer(CurrentPeer, State=#state{tid=TID}) ->
                             connected => ConnectedAddrs,
                             nat_type => State#state.nat_type,
                             network_id => NetworkID,
-                            associations => Associations},
+                            associations => Associations,
+                            timestamp => erlang:system_time(millisecond) + StaleTime},
```

Comment:
my goal here is: to set this in the future so that peers do not look stale all the time. Otherwise they are set to `erlang:system_time(millisecond)` by default in `libp2p_peer:from_map`.

Reply:
I have no idea if I found a bug or I am just miss understanding the `stale_time` stuff

Reply:
forget this I did misunderstand stale time